### PR TITLE
Core/Spells: Scripted some Shaman Spells:

### PR DESCRIPTION
**Pull request by mik1893**
- Ancestral Resolve
- Unleash Elements
- Spirit Link

### CODE REVIEW

**Comment by DDuarte** [comment]
Can this be used on 6.x?

**Comment by Takenbacon** [comment]
@DDuarte Spirit Link, yes. The rest, no.

File: src/server/scripts/Spells/spell_shaman.cpp [hunk 1]
```diff
@@ -1230,10 +1357,99 @@ class spell_sha_tidal_waves : public SpellScriptLoader
...
+                    case ENCHANTMENT_ROCKBITER: // Rockbiter Weapon
+                        unleashSpell = SPELL_SHAMAN_UNLEASH_EARTH; // Unleash Earth
+                        break;
+                    case ENCHANTMENT_WINDFURY: // Windfury Weapon
+                        unleashSpell = SPELL_SHAMAN_UNLEASH_WIND; // Unleash Wind
+                        break;
+                }
+
+                if (hostileSpell && !hostileTarget)
+                    return; // don't allow to attack non-hostile targets. TODO: check this before cast
```
Comment (funjoker) [inline src/server/scripts/Spells/spell_shaman.cpp]: stolen from wowsource ;)

Comment (Warpten) [inline src/server/scripts/Spells/spell_shaman.cpp]: wat

**Comment by Carbenium** [comment]
@mik1893: Any news on this?

**Comment by mik1893** [comment]
sorry guys, i'm back on the line.
Moved to London and things have been going shit.
Should we drop this and start doing good stuff on WoD? :)

**Comment by DDuarte** [comment]
Yes please :)

**Pull request closed without merging**